components/contributors/GithubActivity.tsx

```diff
@@ -15,6 +15,15 @@ import { format } from "date-fns";
 import GithubDiscussion from "../discussions/GithubDiscussion";
 import { IoIosChatboxes } from "react-icons/io";
 import Link from "next/link";
+import { atomWithStorage } from "jotai/utils";
+import { useAtom } from "jotai";
+import { useEffect } from "react";
+
+// atom used for caching
+const activityTypesAtom = atomWithStorage<Activity["type"][]>(
+  "activityTypes",
+  [],
```

Comment:
Default value to be all activities type included.

```suggestion
  [...ACTIVITY_TYPES],
```